server/src/main/java/org/candlepin/guice/CandlepinModule.java

```diff
@@ -418,6 +418,8 @@ private void configureAsyncJobs() {
         JobManager.registerJob(TestJob1.JOB_KEY, TestJob1.class);
         JobManager.registerJob(ExportJob.JOB_KEY, ExportJob.class);
         JobManager.registerJob(ImportJob.JOB_KEY, ImportJob.class);
+        JobManager.registerJob(org.candlepin.async.tasks.RefreshPoolsJob.JOB_KEY,
```

Comment:
Should we import the class like we did for the others?

Reply:
The reason I used the full name is that the pinsetter RefreshPoolsJob is also imported in this file for a guice `bind()`. I suppose I can completely remove that since we no longer need it, and import the new class.

Reply:
Fixed.